spec/octokit/client/repositories_spec.rb

```diff
@@ -381,7 +388,16 @@
 
     describe ".protect_branch", :vcr do
       it "protects a single branch" do
-        branch = @client.protect_branch(@repo.full_name, "master", accept: preview_header)
+        branch = @client.protect_branch(@repo.full_name, "master",
+                                        { accept: preview_header,
+                                          required_status_checks: {
+                                            strict: true,
+                                            contexts: []
+                                          },
+                                          enforce_admins: true,
+                                          required_pull_request_reviews: nil,
+                                        }
```

Comment:
Could we reuse the `protect_params` hash created above? It looks like they are the same thing.

Reply:
That last test is in a different block for some reason. Probably worth reorganizing at some point.